packages/zoe/src/contractFacet/types-ambient.d.ts

```diff
@@ -15,7 +15,10 @@ type ZCFMakeEmptySeatKit = (exit?: ExitRule | undefined) => ZcfSeatKit;
  * synchronously from within the contract, and usually is referred to
  * in code as zcf.
  */
-type ZCF<CT extends unknown = Record<string, unknown>> = {
+type ZCF<
+  CT extends unknown = Record<string, unknown>,
+  IKR extends Record<Keyword, AssetKind> = Record<Keyword, any>,
```

Comment:
I hope you'll add some docs.

(again, it sure would be nice to be able to re-use the examples from the tests as docs)

Reply:
I couldn't get a type annotated example to play nicely (js doc comment inception), but included some documentation additions in [fd4fb78](https://github.com/Agoric/agoric-sdk/pull/9330/commits/fd4fb78372b35f99042a6a7fd9daf5efeee8ae33) and [b0e7088](https://github.com/Agoric/agoric-sdk/pull/9330/commits/b0e7088b58c3562a7a77c36b5d8a8d57c92aba66)